packages/identity/README.md

```diff
@@ -125,7 +125,7 @@ import { Identity } from "@semaphore-protocol/identity"
 const message = "message"
 const identity = new Identity()
 
-const signature = identity.signMessage("message", signature)
+const signature = identity.signMessage("message")
```

Comment:
```suggestion
const signature = identity.signMessage(message)
```

What about using this for consistency?

Reply:
Hey @csiejimmyliu! I just saw that you requested my review, did you push the changes?

Reply:
I pushed the changes. Thanks for the advice!